index.d.ts

```diff
@@ -39,18 +39,18 @@ export interface PackageJson {
   /**
    * npm config values for publish time. like setting an alternate registry
    */
-  publishConfig?:ObjectOfStrings;
+  publishConfig?:Record<string, string>;
 }
 
 // this is what you get from the npm api.
 export interface Packument {
   name: string;
   readme?: string;
   description?: string;
-  'dist-tags': {latest?: string}&ObjectOfStrings;
-  versions: {[key: string]: PackumentVersion};
+  'dist-tags': {latest?: string}&Record<string, string>;
+  versions: Record<string, PackumentVersion>;
   maintainers: Maintainer[];
-  time: {modified: string, created: string, [key: string]: string};
+  time: Record<string, string> & {modified: string; created: string};
```

Comment:
My TS-fu is a little weak here, but could this be typed like this:
```suggestion
  time: {modified: string; created: string, [key: string]: string};
```

(Not a blocker, though)